meal-planning/src/components/shopping-list/ShoppingList.js

```diff
@@ -7,68 +7,57 @@ import Checkbox from '@material-ui/core/Checkbox';
 import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
-import styles from "./ShoppingList.module.scss";
+import styles from './ShoppingList.module.scss';
+import RecipeService from '../../services/RecipeService';
 
 const ShoppingList = () => {
-  const [checked, setChecked] = React.useState([-1]);
-  const ingredients = ['2 bananas', '1/2 cup flour', '1 tsp salt', '1 tsp vanilla', '3 eggs'];
+  const ingredients = RecipeService.getIngredients();
   const [currentList, setList] = React.useState(ingredients);
 
-  const handleToggle = value => () => {
-    const currentIndex = checked.indexOf(value);
-    const newChecked = [...checked];
-
-    if (currentIndex === -1) {
-      newChecked.push(value);
-    } else {
-      newChecked.splice(currentIndex, 1);
-    }
-
-    setChecked(newChecked);
-  };
-
-  const handleDelete = (value) => () => {
-    const currentIndex = currentList.indexOf(value);
-    const newList = [...currentList];
-    newList.splice(currentIndex, 1);
+  const handleDelete = id => () => {
+    const newList = currentList.filter(i => i.id != id);
```

Comment:
```suggestion
    const newList = currentList.filter(i => i.id !== id);
```
should fix eslint warning